xmodule/assets/video/_display.scss

```diff
@@ -271,23 +271,23 @@ $cool-dark: rgb(79, 89, 93); // UXPL cool dark
     }
 
     .closed-captions.is-visible {
-      max-height: ($baseline * 3);
-      border-radius: ($baseline / 5);
-      padding: 8px ($baseline / 2) 8px ($baseline * 1.5);
+      max-height: calc((var(--baseline) * 3));
+      border-radius: calc((var(--baseline) / 5));
+      padding: 8px calc((var(--baseline) / 2)) 8px calc((var(--baseline) * 1.5));
       background: rgba(0, 0, 0, 0.75);
-      color: $yellow;
+      color: var(--yellow);
 
       &::before {
         position: absolute;
         display: inline-block;
         top: 50%;
 
-        @include left($baseline);
+        @include left(var(--baseline));
 
         margin-top: -0.6em;
         font-family: 'FontAwesome';
         content: "\f142";
-        color: $white;
+        color: var(--white);
```

Comment:
This line resolves incorrectly in the built CSS:

```css
color: var(--#fff)
```

Instead of 

```css
color: var(--white);
// or
color: #fff;
```